src/examples/chord/chord.ts

```diff
@@ -0,0 +1,240 @@
+import * as d3 from 'd3'
+import { formatType, handleErrors, log } from '../common/utils'
+
+import {
+  Looker,
+  LookerChartUtils,
+  VisualizationDefinition,
+  VisData
+} from '../types/types'
+
+// Global values provided via the API
+declare var looker: Looker
+declare var LookerCharts: LookerChartUtils
+
+interface ChordVisualization extends VisualizationDefinition {
+  svg?: any,
+  tooltip?: any,
+  computeMatrix: (data: VisData, dimensions: string[], measure: string) => any,
+  titleText: (lookup: any, source: any, target: any, formatter: any) => string,
+}
```

Comment:
Can we do any better than the `any` types here? Just for the sake of TypeScript correctness.